components/script/dom/dommatrixreadonly.rs

```diff
@@ -839,7 +837,7 @@ impl DOMMatrixReadOnlyMethods for DOMMatrixReadOnly {
                 )
             }
         };
-        if self.is2D() {
+        let string = if self.is2D() {
```

Comment:
We need to preserve the specification related comments as [discussed here](https://servo.zulipchat.com/#narrow/stream/426955-Outreachy/topic/Improve.20code.20linting.20in.20Servo's.20CI/near/476784186) i.e we move the comments about steps 2 and 3 to be just before the `let string = if ..` line.

Reply:
just fixed it